.github/workflows/build-images.yml

```diff
@@ -223,8 +223,14 @@ jobs:
         run: |
           rm -rf "scripts/ci"
           mv "main-airflow/scripts/ci" "scripts"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+          cache: 'pip'
+      - run: pip3 install --editable ./dev/breeze
+      - run: python3 -m pytest -n auto --color=yes
```

Comment:
```suggestion
```

We do not really nee to run pytest here :)  -> it's only needed to run `breeze` testing in dedicated action,